Can't install the plugin because of linker error

``` bash
npm install -g homebridge-platform-rcswitch

> homebridge-platform-rcswitch@0.1.48 install /usr/local/lib/node_modules/homebridge-platform-rcswitch
> node-gyp rebuild

gyp WARN EACCES user "root" does not have permission to access the dev dir "/root/.node-gyp/0.12.6"
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/homebridge-platform-rcswitch/.node-gyp"
make: Entering directory '/usr/local/lib/node_modules/homebridge-platform-rcswitch/build'
  CC(target) Release/obj.target/rsswitch/rcswitch.o
cc1: warning: command line option '-std=c++11' is valid for C++/ObjC++ but not for C
../rcswitch.c: In function 'rcswitch_receive_protocol1':
../rcswitch.c:78:18: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (i = 1; i< changeCount; i += 2) {
                  ^
../rcswitch.c: In function 'rcswitch_receive_protocol2':
../rcswitch.c:106:18: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
     for (i = 1; i<changeCount ; i += 2) {
                  ^
../rcswitch.c: In function 'rcswitch_send_scode_word':
../rcswitch.c:135:13: warning: format '%d' expects a matching 'int' argument [-Wformat=]
             fprintf(stderr,"wiringPiSetup failed %d\n");
             ^
  CXX(target) Release/obj.target/rsswitch/rcmodule.o
../rcmodule.cpp: In function 'void SnifferAsync(uv_work_t*)':
../rcmodule.cpp:31:12: warning: unused variable 'isolate' [-Wunused-variable]
  Isolate * isolate = Isolate::GetCurrent();
            ^
../rcmodule.cpp: In function 'void SetupSniffer(const v8::FunctionCallbackInfo<v8::Value>&)':
../rcmodule.cpp:55:11: warning: unused variable 'isolate' [-Wunused-variable]
  Isolate* isolate = args.GetIsolate();
           ^
  SOLINK_MODULE(target) Release/obj.target/rsswitch.node
/usr/bin/ld: cannot find -luv
collect2: error: ld returned 1 exit status
rsswitch.target.mk:125: recipe for target 'Release/obj.target/rsswitch.node' failed
make: *** [Release/obj.target/rsswitch.node] Error 1
make: Leaving directory '/usr/local/lib/node_modules/homebridge-platform-rcswitch/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Linux 4.1.17-v7+
gyp ERR! command "node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/homebridge-platform-rcswitch
gyp ERR! node -v v0.12.6
gyp ERR! node-gyp -v v3.3.0
gyp ERR! not ok 
npm ERR! Linux 4.1.17-v7+
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "homebridge-platform-rcswitch"
npm ERR! node v0.12.6
npm ERR! npm  v3.7.4
npm ERR! code ELIFECYCLE

npm ERR! homebridge-platform-rcswitch@0.1.48 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the homebridge-platform-rcswitch@0.1.48 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the homebridge-platform-rcswitch package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs homebridge-platform-rcswitch
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls homebridge-platform-rcswitch
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /var/homebridge/npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -9,9 +9,10 @@ RCSwitch plugin for the awesome  [Homebridge](https://github.com/nfarina/homebri
 
 # Installation
 
-1. Install homebridge using: npm install -g homebridge
-2. Install this plugin using: npm install -g homebridge-platform-rcswitch
-3. Update your configuration file. See the sample below.
+1. Install libuv-dev using: `apt-get install libuv-dev`
+2. Install homebridge using: `npm install -g homebridge`
+3. Install this plugin using: `npm install -g homebridge-platform-rcswitch`
+4. Update your configuration file. See the sample below.
 
 # Configuration
 

